Signal/src/call/PeerConnectionClient.swift

```diff
@@ -423,10 +423,7 @@ class PeerConnectionClient: NSObject, RTCPeerConnectionDelegate, RTCDataChannelD
         AssertIsOnMainThread()
         Logger.debug("\(TAG) in \(#function)")
 
-        delegate.peerConnectionClient(self, didUpdateLocal: nil)
-        delegate.peerConnectionClient(self, didUpdateRemote: nil)
-
-        PeerConnectionClient.signalingQueue.async {
+        PeerConnectionClient.signalingQueue.sync {
```

Comment:
I believe this is the most significant element of these changes: we block until the pcc's termination logic is complete.